src/styles/index.css

```
@@ -0,0 +1,23 @@
+.headline {
+    font-weight: bold;
+    font-size: 48px;
+    color: #2C358F;
+    margin: 0 20px 0 60px;
+}
+
+.headline-description {
+    font-size: 20px;
+    color: #000000;
+    max-width: 450px;
```

It would be good to increase the width a bit on the description
```suggestion
    width: 800px;
```